src/csync/csync.h

```diff
@@ -103,7 +103,7 @@ Q_ENUM_NS(csync_status_codes_e)
   * Instruction enum. In the file traversal structure, it describes
   * the csync state of a file.
   */
-enum csync_instructions_e {
+enum _csync_instructions_e {
```

Comment:
maybe CSyncInstruction.  Type are camel case in the Qt coding style.

upercase for the first character means member in the owncloud coding style.